src/osa/scripts/closer.py

```diff
@@ -154,32 +158,38 @@ def ask_for_closing():
 def post_process(seq_tuple):
     """Set of last instructions."""
     seq_list = seq_tuple[1]
+    
+    if dl1_datacheck_longterm_file_exits() and not options.test:
+        create_longterm_symlink()
```

Comment:
I think this can be checked at the very beginning of the closer main, not here in `post_process`

Reply:
Right, right, did not realize that.